packages/webpack-cli/bin/cli.js

```diff
@@ -20,6 +20,8 @@ if (packageExists('webpack')) {
 } else {
     promptInstallation('webpack', () => {
         error(`It looks like ${yellow('webpack')} is not installed.`);
-    });
+    })
+        .then(() => info(`${yellow('webpack')} was installed sucessfully.`))
+        .catch(() => error(`Action Interrupted, Please try once again or install ${yellow('webpack')} manually.`));
```

Comment:
What is the error code? if we in `catch`?

Reply:
I think it should be `2`, updated.